en_US/LC_MESSAGES/django.po

```diff
@@ -8718,71 +8669,164 @@ msgstr "An Error Occurred"
+msgid "Explore resources to help you with your contributions."
+msgstr "Explore resources to help you with your contributions."
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:140 kitsune/sumo/jinja2/includes/common_macros.html:318 kitsune/wiki/jinja2/wiki/document.html:79
+#, fuzzy
+msgid "Download Firefox"
+msgstr "MozillaZine"
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:165
+#, fuzzy
+msgid "Manage your information and see your contributions."
+msgstr "Manage accounts and settings"
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:169 kitsune/users/jinja2/users/edit_profile.html:4
+#, fuzzy
+msgid "Edit Profile"
+msgstr "Edit Article"
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:170
+#, fuzzy
+msgid "My Questions"
+msgstr "Question date"
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:178
+msgid "Settings"
+msgstr "Settings"
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:180
+#, fuzzy
+msgid "Manage your privacy and notification preferences."
+msgstr "Email address:"
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:191
+#, fuzzy
+msgid "Exchange private messages with other contributors."
+msgstr "Remove user from group"
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:205 kitsune/users/jinja2/users/profile.html:187
+#, fuzzy
+msgid "Sign Out"
+msgstr "Log Out"
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:253
+#, fuzzy
+msgid "For Contributors"
+msgstr "How to Contribute"
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:267
+#, fuzzy
+msgid "L10N Discussions"
+msgstr "Discussion"
 
-#: kitsune/sumo/jinja2/includes/common_macros.html:85
+#: kitsune/sumo/jinja2/includes/common_macros.html:269
 #, fuzzy
 msgid "Article Discussions"
 msgstr "Discussion"
 
-#: kitsune/sumo/jinja2/includes/common_macros.html:87
+#: kitsune/sumo/jinja2/includes/common_macros.html:272
 #, fuzzy
-msgid "Localization Metrics"
-msgstr "Group Leaders"
+msgid "Moderate Forum Content"
+msgstr "Translate Content"
 
-#: kitsune/sumo/jinja2/includes/common_macros.html:89
+#: kitsune/sumo/jinja2/includes/common_macros.html:279
 #, fuzzy
-msgid "Support Forum Home"
+msgid "Dashboards"
+msgstr "Localizer Dashboard"
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:283
+#, fuzzy
+msgid "Contributor Forums"
 msgstr "Contributor Dashboard"
 
-#: kitsune/sumo/jinja2/includes/common_macros.html:92
+#: kitsune/sumo/jinja2/includes/common_macros.html:322 kitsune/wiki/jinja2/wiki/document.html:84
 #, fuzzy
-msgid "Moderate Forum Content"
-msgstr "Translate Content"
+msgid "What's New"
+msgstr "Ask this"
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:332
+#, fuzzy
+msgid "Give Firefox a tune up"
+msgstr "Firefox Support"
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:334 kitsune/wiki/jinja2/wikiparser/hook_refresh_button.html:1
+#, fuzzy
+msgid "Refresh Firefox"
+msgstr "SpreadFirefox"
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:338
+#, fuzzy
+msgid "How does it work?"
+msgstr "How to"
```

Comment:
If this is a UI, should it be How To (title style) or is this followed by a topic?

Reply:
Similar here.

Reply:
similar here.